src/Fixer/Basic/BracesFixer.php

```diff
@@ -403,7 +456,13 @@ function ($item) {
             }
 
             if ($token->isGivenKind($classyTokens) && !$tokensAnalyzer->isAnonymousClass($index)) {
-                $tokens->ensureWhitespaceAtIndex($startBraceIndex - 1, 1, $this->whitespacesConfig->getLineEnding().$indent);
+                if ($this->configuration['position_after_functions_and_oop_constructs'] === self::LINE_NEXT) {
+                    $ensuredWhitespace = $this->whitespacesConfig->getLineEnding().$indent;
+                } elseif ($this->configuration['position_after_functions_and_oop_constructs'] === self::LINE_SAME) {
```

Comment:
The option accepts only two values, so I would use a simple `else` instead of `elseif` here.